prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2515,7 +2515,7 @@ presubmits:
     - release-24.1
     agent: jenkins
     always_run: false
-    optional: false
```

Comment:
on line 2509 I do not think we should allow to run main on this release branch !

Reply:
Is there a way to configure that? Right now if I do what I intended to do, it will drop too many tests from main, leaving only one. @lentzi90

Reply:
I am confused with running main on metal3-io/ironic-image/release-24.1 `I think metal3-centos-e2e-integration-test-release-1-7` would be enough

Reply:
Nah, I talked myself to confusing. 

Main is tested with main+main, and 24.1 is tested with 1.7+1.7. No issue. Fixed,